govuk-one-login/authentication-api
# ATO-981: set is new account in orch session

## What:

- Sets the value of isNewAccount in the Orch session at the same point as it is set in the regular session
- This includes updating:
   -  Authentication Callback Handler to update this using Auth's userinfo response
   - Updates the AuthCodeGenerationService to update the Orch session alongside the redis session
## How to review
- Code review commit-by-commit
- Deploy to orch dev environment
    - Run through a journey 
    - Check all is well
    - Ran auth acceptance tests against deployed env (sandpit), tests passed
 - Once deployed:
     - Run through journey in staging as feature will be enabled 
     - Verify that in the case of a return code this value is set  
     - Check doc app journey
     - Enable feature flag in higher environments in future PRs

## Checklist





## Related PRs
